src/react-app-env.d.ts

```diff
@@ -1 +1,4 @@
 /// <reference types="react-scripts" />
+REACT_APP_AUTH0_DOMAIN=dev-dvejh6xd.us.auth0.com
```

Comment:
you can't add keys to the repo. they should go on .env, which is currently being ignored by git. it's dangerous if secret keys are leaked. once they are on the history, they are still going to be there even if you remove them on a commit. so it's good to always double check that you're not leaking any secrets.

Reply:
I think there is a way to remove them from the history, but I don't know how to do it

Reply:
yeah this was a mistake, commited it by accident - totally forgot

Reply:
No problem! I don't think it's a big deal here, we're not dealing with real user data and they don't have my credit card haha